src/main/java/io/appform/dropwizard/actors/observers/ObserverContext.java

```diff
@@ -0,0 +1,11 @@
+package io.appform.dropwizard.actors.observers;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class ObserverContext {
+    String opType;
+    String queueName;
+}
```

Comment:
Should we add some details about queue shard (shardId or something else) or sideline queue?